bin/cli/src/lib/write-ui-env-file.ts

```diff
@@ -34,7 +34,7 @@ export async function writeUiEnvFile(stage, local = false) {
     VITE_COGNITO_USER_POOL_CLIENT_ID: deploymentOutput.userPoolClientId,
     VITE_COGNITO_USER_POOL_CLIENT_DOMAIN: deploymentOutput.userPoolClientDomain,
     VITE_COGNITO_REDIRECT_SIGNIN: local
-      ? `"http://localhost:5000/"`
+      ? `"http://localhost:5000/dashboard"`
       : deploymentOutput.applicationEndpointUrl,
```

Comment:
I think you will have to adjust this to be changed as well
```suggestion
      : `${deploymentOutput.applicationEndpointUrl}/dashboard`,
```

Reply:
Correct me if I'm wrong on this. I may be missing context